frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -315,7 +315,49 @@ common_settings.document = {
                     end,
                 },
 
-            }
+            }		
+        },        
+	{
+            text = _("Action on highlight"),
+            sub_item_table = {
+                {
+                    text = _("Default"),
+                    checked_func = function()
+                        local setting = G_reader_settings:readSetting("default_highlight_action")
+                        return setting == "ask" or setting == nil
+                    end,
+                    callback = function()
+                        G_reader_settings:saveSetting("default_highlight_action", "ask")
```

Comment:
Ideally, you could just use `nil`, no real need for `"ask"`.
`G_reader_settings:saveSetting("default_highlight_action", nil)` will remove the setting and make things cleaner.